DigitalLearningSolutions.Web.Tests/Controllers/Register/RegisterAdminControllerTests.cs

```diff
@@ -73,113 +76,27 @@ public void IndexGet_with_invalid_centreId_param_shows_notfound_error()
-            A.CallTo(() => centresDataService.GetCentreAutoRegisterValues(DefaultCentreId))
-                .MustHaveHappened(1, Times.Exactly);
-            A.CallTo(() => userDataService.GetAdminUsersByCentreId(DefaultCentreId)).MustHaveHappened(1, Times.Exactly);
-            result.Should().BeRedirectToActionResult().WithControllerName("LearningSolutions")
-                .WithActionName("AccessDenied");
-        }
-
-        [Test]
-        public void IndexGet_with_centre_autoregisteremail_empty_shows_AccessDenied_error()
-        {
-            // Given
-            A.CallTo(() => centresDataService.GetCentreName(DefaultCentreId)).Returns("Some centre");
-            A.CallTo(() => centresDataService.GetCentreAutoRegisterValues(DefaultCentreId))
-                .Returns((false, string.Empty));
-            A.CallTo(() => userDataService.GetAdminUsersByCentreId(DefaultCentreId)).Returns(new List<AdminUser>());
-
-            // When
-            var result = controller.Index(DefaultCentreId);
-
-            // Then
-            A.CallTo(() => centresDataService.GetCentreName(DefaultCentreId)).MustHaveHappened(1, Times.Exactly);
-            A.CallTo(() => centresDataService.GetCentreAutoRegisterValues(DefaultCentreId))
-                .MustHaveHappened(1, Times.Exactly);
-            A.CallTo(() => userDataService.GetAdminUsersByCentreId(DefaultCentreId)).MustHaveHappened(1, Times.Exactly);
-            result.Should().BeRedirectToActionResult().WithControllerName("LearningSolutions")
-                .WithActionName("AccessDenied");
-        }
-
-        [Test]
-        public void IndexGet_with_centre_with_active_centre_manager_shows_AccessDenied_error()
-        {
-            // Given
-            A.CallTo(() => centresDataService.GetCentreName(DefaultCentreId)).Returns("Some centre");
-            A.CallTo(() => centresDataService.GetCentreAutoRegisterValues(DefaultCentreId))
-                .Returns((false, "email@email"));
-
-            var centreManagerAdmin = new AdminUser { CentreId = DefaultCentreId, IsCentreManager = true };
-            A.CallTo(() => userDataService.GetAdminUsersByCentreId(DefaultCentreId))
-                .Returns(new List<AdminUser> { centreManagerAdmin });
-
-            // When
-            var result = controller.Index(DefaultCentreId);
-
-            // Then
-            A.CallTo(() => centresDataService.GetCentreName(DefaultCentreId)).MustHaveHappened(1, Times.Exactly);
-            A.CallTo(() => centresDataService.GetCentreAutoRegisterValues(DefaultCentreId))
-                .MustHaveHappened(1, Times.Exactly);
-            A.CallTo(() => userDataService.GetAdminUsersByCentreId(DefaultCentreId)).MustHaveHappened(1, Times.Exactly);
-            result.Should().BeRedirectToActionResult().WithControllerName("LearningSolutions")
-                .WithActionName("AccessDenied");
-        }

```

Comment:
I deleted these because these were testing the various cases when the current `RegisterAdminHelper.IsRegisterAdminAllowed` returns false, and I check this cases in `RegisterAdminHelperTests`.

As you noticed in another comment, I was planning to add a test that check the result is `AccessDenied` when `RegisterAdminHelper.IsRegisterAdminAllowed` returns false, but forgot.

Reply:
Ok